flintrock/flintrock.py

```diff
@@ -268,6 +274,8 @@ def launch(
         elif spark_git_commit:
             print("Warning: Building Spark takes a long time. "
                   "e.g. 15-20 minutes on an m3.xlarge instance on EC2.")
+            if spark_git_commit == "latest":
+                spark_git_commit = get_last_commit_sha(spark_git_repository)
```

Comment:
I'd say leave it out for starters and keep things simple. We can easily add it in later. :+1: 

And I agree: If you care about the commit, you probably already looked it up beforehand.